rebac-admin-backend/v1/identities.go

```diff
@@ -0,0 +1,176 @@
+// Copyright 2024 Canonical Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+package v1
+
+import (
+	"encoding/json"
+	"net/http"
+
+	r "github.com/canonical/identity-platform-admin-ui/rebac-admin-backend/v1/resources"
```

Comment:
let's not do that.. it makes reading code more confusing than it needs to be.. just use resources.GetIdentitiesResponse etc..

Reply:
also.. should the wireformats live in their own package so any potential clients can import them independently of the handler implementation package?

Reply:
Wireformats are now in the same package but different file, we could move them to a different package, I'll track this so we can definitely reason about it. Thanks!

Reply:
@alesstimec The handlers are already implemented in the library's root package (i.e., not in the `resources` package).